remote_attestation/java/src/com/google/oak/remote_attestation/ClientAttestationEngine.java

```diff
@@ -104,48 +104,49 @@ public ClientHello createClientHello() throws IllegalStateException {
     new Random().nextBytes(random);
 
     // Create client hello message.
-    ClientHello clientHello =
-        ClientHello.newBuilder().setRandom(ByteString.copyFrom(random)).build();
+    Message.ClientHello clientHello = new Message.ClientHello(random);
 
     // Update current transcript.
-    byte[] serializedClientHello = clientHello.toByteArray();
+    byte[] serializedClientHello = clientHello.serialize();
     transcript = Bytes.concat(transcript, serializedClientHello);
```

Comment:
May be useful to have a method to append to the transcript, since it's quite common.